workflow/snakemake_rules/nextclade.smk

```diff
@@ -62,52 +62,94 @@ if config.get("s3_dst") and config.get("s3_src"):
     ruleorder: download_nextclade_tsv_from_s3 > create_empty_nextclade_info
     ruleorder: download_previous_alignment_from_s3 > create_empty_nextclade_aligned
 
+    def _convert_dataset_name(wildcards):
+        if wildcards.reference == '':
+            dataset_name="sars-cov-2"
+        elif wildcards.reference == '_21L':
+            dataset_name="sars-cov-2-21L"
+        else:
+            # We shouldn't run into this since we have wildcard_constraints,
+            # but doesn't hurt to include it in case that changes
+            raise ValueError(f"Cannot convert unsupported reference {wildcards.reference!r} to Nextclade dataset name")
+
+        return f"data/nextclade_data/{dataset_name}.zip",
+
+    rule use_nextclade_cache:
+        input:
+            nextclade="./nextclade",
```

Comment:
OH, just ran into an error with this after removing the local executable:

```shell
rm ./nextclade
nextstrain build --envdir ~/Repos/env.d/aws/ --image nextstrain/ncov-ingest . data/genbank/use_nextclade_cache.txt --configfile config/genbank.yaml --forceall
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
Building DAG of jobs...
Relative file path './nextclade' starts with './'. This is redundant and strongly discouraged. It can also lead to inconsistent results of the file-matching approach used by Snakemake. You can simply omit the './' for relative file paths.
MissingInputException in rule use_nextclade_cache in file /nextstrain/build/workflow/snakemake_rules/nextclade.smk, line 66:
Missing input files for rule use_nextclade_cache:
    output: data/genbank/use_nextclade_cache.txt
    wildcards: reference=
    affected files:
        ./nextclade
```

The reason I'm using `./nextclade` is to ensure `use-nextclade-cache` the locally downloaded executable instead of the nextclade command available in the environment.

To avoid further mix-up here, I'm just going to move the executable output to `data/nextclade`.

Reply:
Updated in https://github.com/nextstrain/ncov-ingest/pull/466/commits/9a2ca5759a4772ba75b4c9079d075172d813bb4e